submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
```

Comment:
`createElem...` ? =)
![image](https://user-images.githubusercontent.com/33497965/187794458-527bf075-32dd-4f34-affd-c9c6d641de5d.png)

You don't pay for letter, words, so give proper names for functions variables etc. 
You should understand that you write code for humans not for machines.

Reply:
I will try to use more meaningful and understandable names for variables and functions in the future. Previously did not pay close attention to it. Thanks for the thoughtful note.